lib/ansible_tower_client/inventory_source.rb

```diff
@@ -0,0 +1,21 @@
+module AnsibleTowerClient
+  class InventorySource < BaseModel
+    def self.endpoint
+      "inventory_sources".freeze
+    end
+
+    def can_update?
```

Comment:
@bdunne - I liked how can_update? read on https://github.com/syncrou/ansible_tower_client/blob/a5d91334e8ee883f008ee8a4a1462ca86f6501a3/lib/ansible_tower_client/inventory.rb?#L13 in terms of thestatement:  If inventory source can update?